assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -62,6 +64,10 @@ function MultiSelectionMovable({ selectedElements }) {
     }
   }, [selectedElements, moveable, nodesById]);
 
+  // ⌘ key disables snapping
+  useGlobalKeyDownEffect('meta', () => setCanSnap(false), [setCanSnap]);
```

Comment:
Hmm. The global handler here is probably ok, but is there no other way to bound it at all?

Reply:
Only ref we have here is `moveable` which is not a Node

Reply:
Would `pageContainer` from `useCanvas()` work here?

Reply:
That does not work unfortunately :/ pageContainer is below the frames layer so events aren't captured there. We need a ref to the frames layer but not sure if adding it is worth it here. Other option is to have the snapping state be held at the frames layer and passed down, wdyt?

Reply:
Let's move the way you have for now. We'll see if we run into conflicts.